platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/utils/MathUtils.java

```diff
@@ -0,0 +1,79 @@
+package com.mapbox.mapboxsdk.utils;
+
+// TODO Remove this class if we finally include it within MAS 3.x (GeoJSON)
```

Comment:
Remove or ticket.

Reply:
Ticketed

Reply:
@electrostat 
> Ticketed

This should be ticketed in this repo 👉 https://github.com/mapbox/mapbox-gl-native/issues

Reply:
Follow up ticket 👉 https://github.com/mapbox/mapbox-gl-native/issues/11182